packages/projection/test/operators/Mappers/withHandles.test.ts

```diff
@@ -1,7 +1,9 @@
+/* eslint-disable sort-imports-es6-autofix/sort-imports-es6 */
```

Comment:
This looks suspicious, is there a good reason to disable these rules?

Reply:
addressed 8fe85a39e79855d7b78fb18836b808b1d6b2d5a8